### PR TITLE
Chart: 1.5.0 changelog

### PR DESCRIPTION
**Pull request by ephraimbuddy**
Add changelog for chart version 1.5.0.

### CODE REVIEW

File: chart/CHANGELOG.txt [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+
+New Features
+""""""""""""
+
+- Add support for custom command and args in jobs (#20864)
+- Support for priorityClassName (#20794)
+- Add envFrom to the flower deployment (#21401)
+- Add annotations to cleanup pods (#21484)
+
+
```
Comment (jedcunningham) [inline chart/CHANGELOG.txt]:
```suggestion
```

File: chart/CHANGELOG.txt [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+Improvements
+""""""""""""
+
+- Speedup liveness probe for scheduler and triggerer (#20833, #21108)
+- Update git-sync to v3.4.0 (#21309)
+- Remove default auth backend setting (#21640)
+
```
Comment (jedcunningham) [inline chart/CHANGELOG.txt:18]:
```suggestion
```

File: chart/CHANGELOG.txt [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+
+Bug Fixes
+"""""""""
+- Fix elasticsearch URL when username/password are empty (#21222)
+- Mount `airflow.cfg` in wait-for-airflow-migrations containers (#20609)
+- Grant pod log reader to triggerer serviceaccount (#21111)
+
```
Comment (jedcunningham) [inline chart/CHANGELOG.txt:24]:
```suggestion
```

File: chart/CHANGELOG.txt [hunk 4]
```diff
@@ -1,3 +1,41 @@
...
+""""""""""""""""
+- Simplify chart docs for configuring Airflow (#21747)
+- Add extra information about time synchronization needed (#21685)
+- Fix extra containers docs (#20787)
+
+Misc
+""""
+- Use ``2.2.4`` as default Airflow version (#21745)
+
+
```
Comment (jedcunningham) [inline chart/CHANGELOG.txt]:
```suggestion
```

File: chart/Chart.yaml [hunk 5]
```diff
@@ -64,3 +64,87 @@ annotations:
       url: https://airflow.apache.org/docs/apache-airflow/2.2.4/_images/code.png
     - title: Task Instance Context Menu
       url: https://airflow.apache.org/docs/apache-airflow/2.2.4/_images/context.png
+  artifacthub.io/changes: |
+    - description: Add support for custom command and args in jobs
+      kind: added
+      links:
+      - name: '#20864'
+        url: https://github.com/apache/airflow/pull/20864
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]: We can probably get rid of these extra newlines.

```suggestion
```

**Review by jedcunningham — APPROVED** [review]

File: chart/CHANGELOG.txt [hunk 6]
```diff
@@ -1,3 +1,36 @@
+Airflow Helm Chart 1.5.0, 2022-03-07
+------------------------------------
+
+New Features
+""""""""""""
+
+- Add support for custom command and args in jobs (#20864)
+- Support for priorityClassName (#20794)
+- Add envFrom to the flower deployment (#21401)
```
Comment (jedcunningham) [inline chart/CHANGELOG.txt]:
```suggestion
- Support for ``priorityClassName`` (#20794)
- Add ``envFrom`` to the Flower deployment (#21401)
```

File: chart/CHANGELOG.txt [hunk 7]
```diff
@@ -1,3 +1,36 @@
...
+""""""""""""
+
+- Speedup liveness probe for scheduler and triggerer (#20833, #21108)
+- Update git-sync to v3.4.0 (#21309)
+- Remove default auth backend setting (#21640)
+
+Bug Fixes
+"""""""""
+- Fix elasticsearch URL when username/password are empty (#21222)
+- Mount `airflow.cfg` in wait-for-airflow-migrations containers (#20609)
```
Comment (jedcunningham) [inline chart/CHANGELOG.txt]:
```suggestion
- Mount ``airflow.cfg`` in wait-for-airflow-migrations containers (#20609)
```

File: chart/Chart.yaml [hunk 8]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+      kind: added
+      links:
+      - name: '#20864'
+        url: https://github.com/apache/airflow/pull/20864
+    - description: Support for priorityClassName
+      kind: added
+      links:
+      - name: '#20794'
+        url: https://github.com/apache/airflow/pull/20794
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

Sorry, I should have been more clear, I meant all these as well.

File: chart/Chart.yaml [hunk 9]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+    - description: Add envFrom to the flower deployment
+      kind: added
+      links:
+      - name: '#21401'
+        url: https://github.com/apache/airflow/pull/21401
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

File: chart/Chart.yaml [hunk 10]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+    - description: Add annotations to cleanup pods
+      kind: added
+      links:
+      - name: '#21484'
+        url: https://github.com/apache/airflow/pull/21484
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

File: chart/Chart.yaml [hunk 11]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+    - description: Speedup liveness probe for scheduler and triggerer
+      kind: changed
+      links:
+      - name: '#20833'
+        url: https://github.com/apache/airflow/pull/20833
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

File: chart/Chart.yaml [hunk 12]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+    - description: Update git-sync to v3.4.0
+      kind: changed
+      links:
+      - name: '#21309'
+        url: https://github.com/apache/airflow/pull/21309
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

File: chart/Chart.yaml [hunk 13]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+    - description: Remove default auth backend setting
+      kind: changed
+      links:
+      - name: '#21640'
+        url: https://github.com/apache/airflow/pull/21640
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

File: chart/Chart.yaml [hunk 14]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+    - description: Fix elasticsearch URL when username/password are empty
+      kind: fixed
+      links:
+      - name: '#21222'
+        url: https://github.com/apache/airflow/pull/21222
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

File: chart/Chart.yaml [hunk 15]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+    - description: Mount `airflow.cfg` in wait-for-airflow-migrations containers
+      kind: fixed
+      links:
+      - name: '#20609'
+        url: https://github.com/apache/airflow/pull/20609
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

File: chart/Chart.yaml [hunk 16]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+    - description: Grant pod log reader to triggerer serviceaccount
+      kind: fixed
+      links:
+      - name: '#21111'
+        url: https://github.com/apache/airflow/pull/21111
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

File: chart/Chart.yaml [hunk 17]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+    - description: 'Docs: Simplify chart docs for configuring Airflow'
+      kind: changed
+      links:
+      - name: '#21747'
+        url: https://github.com/apache/airflow/pull/21747
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

File: chart/Chart.yaml [hunk 18]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+    - description: 'Docs: Add extra information about time synchronization needed'
+      kind: changed
+      links:
+      - name: '#21685'
+        url: https://github.com/apache/airflow/pull/21685
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

File: chart/Chart.yaml [hunk 19]
```diff
@@ -64,3 +64,86 @@ annotations:
...
+    - description: 'Docs: Fix extra containers docs'
+      kind: changed
+      links:
+      - name: '#20787'
+        url: https://github.com/apache/airflow/pull/20787
+
```
Comment (jedcunningham) [inline chart/Chart.yaml]:
```suggestion
```

File: chart/Chart.yaml [hunk 20]
```diff
@@ -64,3 +64,79 @@ annotations:
...
+      kind: changed
+      links:
+      - name: '#20787'
+        url: https://github.com/apache/airflow/pull/20787
+    - description: 'Misc: Use ``2.2.4`` as default Airflow version'
+      kind: changed
+      links:
+      - name: '#21745'
+        url: https://github.com/apache/airflow/pull/21745
+    - description: 'Misc: Change redis image to bullseye'
```
Comment (ephraimbuddy) [inline chart/Chart.yaml]:
```suggestion
    - description: 'Misc: Change Redis image to bullseye'
```

File: chart/CHANGELOG.txt [hunk 21]
```diff
@@ -1,3 +1,37 @@
...
+Doc only changes
+""""""""""""""""
+- Simplify chart docs for configuring Airflow (#21747)
+- Add extra information about time synchronization needed (#21685)
+- Fix extra containers docs (#20787)
+
+Misc
+""""
+- Use ``2.2.4`` as default Airflow version (#21745)
+- Change redis image to bullseye (#21875)
```
Comment (ephraimbuddy) [inline chart/CHANGELOG.txt]:
```suggestion
- Change Redis image to bullseye (#21875)
```

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**